File: frontend/src/app/entity/person.ts
export class Person {
  private firstName: string;
  private lastName: string;
  private birthday: string;
  private phoneNumber: string;
  private email: string;
  private isMessageOnBirthday: boolean;

  get FirstName(): string {
    return this.firstName;
  }

  set FirstName(value: string) {
    this.firstName = value;
  }

  get LastName(): string {
    return this.lastName;
  }

  set LastName(value: string) {
    this.lastName = value;
  }

  get Birthday(): string {
    return this.birthday;
  }

  set Birthday(value: string) {
    this.birthday = value;
  }

  get PhoneNumber(): string {
    return this.phoneNumber;
  }

  set PhoneNumber(value: string) {
    this.phoneNumber = value;
  }

  get Email(): string {
    return this.email;
  }

  set Email(value: string) {
    this.email = value;
  }

  get IsMessageOnBirthday(): boolean {
    return this.isMessageOnBirthday;
  }

  set IsMessageOnBirthday(value: boolean) {
    this.isMessageOnBirthday = value;
  }
}

File: backend/src/main/java/com/ys/backend/entity/Person.java
package com.ys.backend.entity;

public class Person {
    private String firstName;
    private String lastName;
    private String birthday;
    private String phoneNumber;
    private String email;
    private boolean isMessageOnBirthday;

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastname) {
        this.lastName = lastname;
    }

    public String getBirthday() {
        return birthday;
    }

    public void setBirthday(String birthday) {
        this.birthday = birthday;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public boolean getIsMessageOnBirthday() {
        return isMessageOnBirthday;
    }

    public void setIsMessageOnBirthday(boolean isMessageOnBirthday) {
        this.isMessageOnBirthday = isMessageOnBirthday;
    }

    @Override
    public String toString() {
        return "Person{" +
                "firstName: '" + firstName + '\'' +
                ", lastName: '" + lastName + '\'' +
                ", birthday: '" + birthday + '\'' +
                ", phoneNumber: '" + phoneNumber + '\'' +
                ", email: '" + email + '\'' +
                ", isMessageOnBirthday: " + isMessageOnBirthday +
                '}';
    }
}